### PR TITLE
refactor(Ga): Refactor georgia

### PR DESCRIPTION
**Pull request by flooie**
Drop old tests that are no longer applicable
Drop duplicated GA HTML tests
Update scraper to parse names better
Sped up scraper to scrape only the last month

### CODE REVIEW

File: juriscraper/opinions/united_states/state/ga.py [hunk 1]
```diff
@@ -15,69 +14,59 @@ class Site(OpinionSiteLinear):
...
-                            "name": titlecase(name.lstrip(" .,")),
-                            "url": item.xpath(".//a[1]/@href")[0],
-                        }
-                    )
+            if date_summary.date() < self.last_month:
+                # GA lists all opinions from the previous year on the same page
+                # but we don't need to scrape them every time.  So cancel out
+                # after 31 days.
+                # This should resolve the speed warning we receive on GA.
+                break
```
Comment (mlissner) [inline juriscraper/opinions/united_states/state/ga.py:59]: This should be left to CL. It's better to give CL lots of information and let it decide when to cancel.

File: juriscraper/opinions/united_states/state/ga.py [hunk 2]
```diff
@@ -15,69 +14,59 @@ class Site(OpinionSiteLinear):
...
+                continue
+            docket = ", ".join(dockets)
+            name = title.split(dockets[-1])[-1].strip("., ")
+            summary = link.xpath(
+                ".//parent::li/parent::ul/preceding-sibling::p[1]"
+            )[0].text_content()
+            # We've got en/em dash and hyphens.
+            date_str = (
+                summary.split("–")[0].split("—")[0].split("-")[0].strip()
+            )
```
Comment (mlissner) [inline juriscraper/opinions/united_states/state/ga.py:49]: An explanation would help a lot here.

File: juriscraper/opinions/united_states/state/ga.py [hunk 3]
```diff
@@ -15,69 +14,59 @@ class Site(OpinionSiteLinear):
...
+            url = link.get("href")
+            title = link.text_content()
+            if not title:
+                continue
+            dockets = re.findall(r"S?\d{2}\w\d{4}", title)
+            if not dockets:
+                # Skip empty links in the HTML
+                continue
+            docket = ", ".join(dockets)
+            name = title.split(dockets[-1])[-1].strip("., ")
```
Comment (mlissner) [inline juriscraper/opinions/united_states/state/ga.py:42]: An example here, please too.

**Review by mlissner — COMMENTED** [review]
A couple small comments.

File: juriscraper/opinions/united_states/state/ga.py (hunk 1 above)
Comment (satsuki-chan) [reply juriscraper/opinions/united_states/state/ga.py]: Removed date restriction for the opinions to scrape.
All opinions published in the year will be scraped, regardless of their publication date.

File: juriscraper/opinions/united_states/state/ga.py (hunk 2 above)
Comment (satsuki-chan) [reply juriscraper/opinions/united_states/state/ga.py:58]: Added examples and comments about the characters used as separator between the dates and the summary text.

File: juriscraper/opinions/united_states/state/ga.py (hunk 3 above)
Comment (satsuki-chan) [reply juriscraper/opinions/united_states/state/ga.py:44]: Added.

**Comment by satsuki-chan** [comment]
Closes #453

**Comment by mlissner** [comment]
Thank you, merged!

**Pull request merged**